### PR TITLE
PXP-519: [CLI] Change the linter rule to accept any path declared in the File.exists/2 function

### PR DESCRIPTION
**Pull request by mfauzaan**
## Summary
Currently, the linter rule expects the path value in the `File.exists/2` function to be the same as the file name passed to the `import_config/1` function. However, in Elixir, the `File.exists?/2` function behaves differently and expects paths relative to the current working directory.

To make the code work properly, we need to change the `File.exists/2` function call to use a path relative to the current working directory, like so:

```
if File.exists?("config/dev.secrets.exs") do
  import_config("dev.secrets.exs")
end
```

This PR updates the linter rule to accept any path declared in the `File.exists/2` function, independent of the value in the `import_config/1` function. This allows developers to use any valid path with the `File.exists/2` function, without breaking the linter rule.

Ticket: [PXP-519](https://mindvalley.atlassian.net/browse/PXP-519)

[PXP-519]: https://mindvalley.atlassian.net/browse/PXP-519?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by onimsha — APPROVED** [review]

**Comment by sauron-droid** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/mindvalley/wukong-cli/pull/92#" title="Author self-approved">mfauzaan</a>*, *<a href="https://github.com/mindvalley/wukong-cli/pull/92#pullrequestreview-1393077829" title="Approved">onimsha</a>*


The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=mindvalley%2Fwukong-cli).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/mindvalley/wukong-cli/blob/main/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
